SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute.Android/Distribute.cs

```diff
@@ -126,6 +128,27 @@ public bool OnReleaseAvailable(Activity activity, AndroidReleaseDetails androidR
                 }
                 return false;
             }
+
+            public void OnNoReleaseAvailable(Activity activity)
+            {
+                if (_noReleaseAvailableCallback != null)
+                {
+                    _noReleaseAvailableCallback();
+                }
+            }
+        }
+
+        static void SetNoReleaseAvailable(NoReleaseAvailableCallback noReleaseAvailable)
```

Comment:
Let's keep the class `Listener` on the bottom and move this method right under the `SetReleaseAvailableCallback` one.